spec/controllers/devise/saml_sessions_controller_spec.rb

```diff
@@ -39,4 +43,33 @@ def destroy
       expect(response).to redirect_to(%r(\Ahttp://localhost:8009/saml/logout\?SAMLRequest=))
     end
   end
+
+  describe '#idp_sign_out' do
```

Comment:
I feel like this should have a context for when `saml_session_index_key` is not configured, that it returns 500.